spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -66,3 +66,6 @@ definitions:
                   values:
                       - 0: System Healthy
                       - 1: An error has occurred
+        - protocol:
```

Comment:
uh, I believe that the intention for the version stuff would go into the reserved field of the heartbeat message?